Shoko.Server/Models/SVR_AnimeSeries.cs

```diff
@@ -395,17 +395,17 @@ public void Populate(SVR_AniDB_Anime anime)
             SeriesNameOverride = string.Empty;
         }
 
-        public void CreateAnimeEpisodes()
+        public void CreateAnimeEpisodes(SVR_AniDB_Anime anime = null)
```

Comment:
Should I revert these changes? I didn't see why we needed to consult the database/cache to get the anime if we already had it present in the scope when the function was called.